packages/figma-plugins/studio/src/ui/App.tsx

```diff
@@ -1,10 +1,14 @@
+import { writeTextToClipboard } from '@opendesign/figma-plugin-common/dist/ui/clipboard'
 import React, { useState, useCallback, useEffect } from 'react'
 
 import './App.css'
 import logoPng from './logo.png'
 import { dispatchToFigma } from './utils/dispatcher'
 import { sleep } from './utils/sleep'
 
+const MESSAGE_SUCCESS = 'Copy was successful, paste into Ceros Studio.'
+const MESSAGE_FAILURE = 'Copying failed❗ Please try again in the desktop app.'
```

Comment:
Is it me or there's emoji instead of excalamation mark? 🤔

Reply:
Is it a problem? Hoped that that it will look better, that user can spot it quicker.. can revert it if objections, just my preference here..